src/ui/citation-popup/view.js

```diff
@@ -21,8 +21,18 @@ const output = document.getElementById("output");
 function main () {
 
     /* State */
+    const json = JSON.parse(inputRefsView.textContent);
```

Comment:
Could you check what happens when there's an error when parsing? You can for example throw an exception from here or set "json" to null. You don't have to display an error message but at least it shouldn't crash the plugin.